azure-mgmt-eventhub/tests/test_azure_mgmt_eventhub_check_name_availability.py

```diff
@@ -0,0 +1,38 @@
+﻿# coding: utf-8
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+import unittest
+
+import azure.mgmt.eventhub.models
+from azure.mgmt.eventhub.models import EHNamespace
+from azure.common.credentials import ServicePrincipalCredentials
+
+from devtools_testutils import AzureMgmtTestCase, ResourceGroupPreparer
+
+
+class MgmtEventHubTest(AzureMgmtTestCase):
+
+    def setUp(self):
+        super(MgmtEventHubTest, self).setUp()
+
+        self.eventhub_client = self.create_mgmt_client(
+            azure.mgmt.eventhub.EventHubManagementClient
+        )
+
+    def process(self, result):
+        pass
+
+    @ResourceGroupPreparer()
```

Comment:
You don't need a ResourceGroupPreparer since you don't use a ResoruceGroup

Reply:
resolved